graphql/resolvers.go

```diff
@@ -972,28 +1005,24 @@ func (r *queryResolver) Projects(ctx context.Context) (*Projects, error) {
 	}
 
 	usr := MustHaveUser(ctx)
-	groupsMap := make(map[string][]*restModel.UIProjectFields)
-	favorites := []*restModel.UIProjectFields{}
+	groupsMap := make(map[string][]*restModel.APIProjectRef)
+	favorites := []*restModel.APIProjectRef{}
 
 	for _, p := range allProjs {
 		groupName := strings.Join([]string{p.Owner, p.Repo}, "/")
-
-		uiProj := restModel.UIProjectFields{
-			DisplayName: p.DisplayName,
-			Identifier:  p.Identifier,
-			Repo:        p.Repo,
-			Owner:       p.Owner,
+		apiProjectRef := restModel.APIProjectRef{}
+		if pErr := apiProjectRef.BuildFromService(p); pErr != nil {
```

Comment:
we usually just use `err` for errors any reason to use `pErr` here?

Reply:
I was trying to avoid shadowing the original `err` but I realized I could achieve that by omitting `:`